### DISCUSSION TITLE
Loading LoRAs via text prompt

### DISCUSSION THREAD

**Discussion by 42degrees** [Comfy-Org/ComfyUI#1686] [Q&A] [answered]
I've been playing with ComfyUI for about a week and I started creating these really complex graphs with interesting combinations of graphs to enable and disable the loras depending on what I was doing.  However, I'm pretty sure I don't need to use the Lora loaders at all since it appears that by putting <lora:[name of file without extension]:1.1> I can load any lora for this prompt.  Note that I started using Stable Diffusion with Automatic1111 so all of my lora files are stored within StableDiffusion\models\Lora and not under ComfyUI.

Am I fooling myself that this is working?  I couldn't find any documentation that confirms that this is a valid way of loading loras or any or the other types of loaders.  Frankly, I'm hoping that I'm not fooling myself because that's a whole lot easier to store off in a file of interesting prompts than storing the whole saved configuration.

I didn't make this up, I saw samples of this in prompts all over civitai.com.  However some of the loras I download from there have terribly complex (and they still often don't have enough information to find the original entry on Civitai).  It's nice to be able to change the title of the loading block to clarify the lora.

As a correlated question, is the order that loras are attached important at all?  Should I be playing with the order to get different results?

Thanks for reading!  Any thoughts would be appreciated as I'm still figuring this all out.

**Comment by cdsama** [comment, marked as answer]
LORA will not be loaded if you do noting on it，try this plugin to automatic load LORA by prompt text.  

[https://github.com/badjeff/comfyui_lora_tag_loader/](https://github.com/badjeff/comfyui_lora_tag_loader/ )

![](https://github.com/badjeff/comfyui_lora_tag_loader/raw/master/workflows/workflow_sample.png)

**Comment by 42degrees** [comment]
Thanks for that.  For the next newbie though, it should be stated that first the Load LoRA Tag has its own multiline text editor.  I could never find a node that simply had the multiline text editor and nothing for output except STRING (the node in that screen shot that has the Title of, "Positive Prompt - Model 1").  Also, to create the CLIP Text Encode that has a text input, you have to right-click on a regular CLIP Text Encode node and choose "Convert text to input".  It took me over 30 minutes to figure that out (I was searching for a node that only did text encoding).  It looks like this:
![image](https://github.com/comfyanonymous/ComfyUI/assets/1676142/c0376d2b-6a6a-454e-afb2-9ef1a2401373)

and then it looks like this when you're done:

![image](https://github.com/comfyanonymous/ComfyUI/assets/1676142/d5381692-537f-443c-9415-43559b6326af)

**Comment by Vinylspider** [comment]
Does this node handle multiple lora's in the prompt ?

**Reply by ltdrdata** [reply]
There are various custom nodes that perform LoRA loading through prompts.

ImpactWildcardEncode is one of them.
